elixir/apps/web/lib/web/components/navigation_components.ex

```diff
@@ -22,10 +22,7 @@ defmodule Web.NavigationComponents do
             <.icon name="hero-bars-3-center-left" class="w-6 h-6" />
             <span class="sr-only">Toggle sidebar</span>
           </button>
-          <a
-            href="https://www.firezone.dev/?utm_source=product"
-            class="flex items-center justify-between mr-4"
-          >
+          <a href={~p"/"} class="flex items-center justify-between mr-4">
```

Comment:
@jamilbk I think there is no point in going to the website for a user who is already using the portal because the website is mostly about marketing/sales. The common practice for logo is to go to `/` on click and we get an additional benefit: it provides a poor man's account switch. If you have multiple accounts you can click on the logo and quickly jump between them.

Reply:
Do we have any other way for them to get to Docs, Support, or the Contact forms website that's easily accessible? This might be the only place.

I think we maybe want to leave this link as-is since those use cases are probably more common than switching accounts (I don't think anyone belongs to more than one account?).

Reply:
That'll require some more styling work. I think this is OK for now.